system/security/LdapSecurity/ldaputils.cpp

```diff
@@ -105,7 +105,40 @@ LDAP* LdapUtils::LdapInit(const char* protocol, const char* host, int port, int
             return nullptr;
         }
 #endif
+
+    }
+
+    //Set TLS KeepAlive options
+    int kaTime;
+    int kaInterval;
+    int kaProbes;
+    if (queryKeepAlive(kaTime, kaInterval, kaProbes))//query OpenLDAP per connection tcp-keepalive settings
+    {
+        DBGLOG("queryKeepAlive returns time=%d, intvl=%d, probes=%d", kaTime, kaInterval, kaProbes);
```

Comment:
skip DBGLOG ?  Perhaps change it to only log one line if any of the keepalive settings are made ?